apps/operate/components/Contracts/hooks.ts

```diff
@@ -55,9 +55,9 @@ const STAKING_CONTRACT_DETAILS: Record<Address, StakingContractDetailsInfo> = {
   },
   '0x00000000000000000000000088996bbde7f982d93214881756840ce2c77c4992': {
     availableOn: 'optimusQuickstart',
-    minOperatingBalance: 10,
+    minOperatingBalance: 0.17,
```

Comment:
maybe could simplify it to just `minOperatingBalance: '~0.17 ETH'` and remove the `minOperatingBalanceToken ` since we don't need to convert the value (sad to delete such a nice code btw)
and maybe `isVariousChain` can be something like a `minOperatingBalanceHint: '(Total Various Chains)'` so the render logic is a bit easier to read... wdyt?